bin/asinit

```diff
@@ -8,6 +8,30 @@ const colors = require("../cli/util/colors");
 const version = require("../package.json").version;
 const options = require("../cli/util/options");
 
+const commands = {
+  "npm": {
+    install: "npm install",
+    run: "npm run",
+    test: "npm test"
+  },
+  "yarn": {
+    install: "yarn install",
```

Comment:
```suggestion
    install: "yarn",
```

Reply:
Both would work, but I felt like using the explicit `install` would help less savvy users that may not know that `yarn` without argument is an alias to `install`

Reply:
I see. ok